src/pylhe/__init__.py

```diff
@@ -225,35 +230,31 @@ def visualize(event, outputname):
     g = nx.DiGraph()
     for i, p in enumerate(event.particles):
         g.add_node(i, attr_dict=p.__dict__)
-        name = pypdt.particle(p.id).name
-        greek = [
-            "gamma",
-            "nu",
-            "mu",
-            "tau",
-            "rho",
-            "Xi",
-            "Sigma",
-            "Lambda",
-            "omega",
-            "Omega",
-            "Alpha",
-            "psi",
-            "phi",
-            "pi",
-            "chi",
-        ]
-        for greekname in greek:
-            if greekname in name:
-                name = name.replace(greekname, "\\" + greekname)
-        if "susy-" in name:
-            name = name.replace("susy-", "\\tilde ")
-        g.node[i].update(texlbl="${}$".format(name))
+        try:
+            iid = int(p.id)
+            if is_SUSY(iid):
```

Comment:
I gave you this workaround for SUSY particles which I work on a extensive data file with "all" particles. This CSV file you access allows one to get mapping between several particle properties without the need to create all `Particle` instances. The mapping works for all particles, so no need to do anything special for SUSY. You can trivially look at the contents of `_PDGID2PDGNameMap` and you will see what I mean …

In other words, no need for the `else`.